### PR TITLE
Install tox with pip --user to avoid permission issues

### PR DESCRIPTION
**Pull request by boris-42**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -30,7 +30,7 @@ jobs:
 
       - run:
           name: Install tox
-          command: pip install tox
+          command: sudo pip install tox
```
Comment (swilly22) [inline .circleci/config.yml:33]: `pip install tox --user` won't do it?

Comment (boris-42) [reply .circleci/config.yml:33]: @swilly22 I tried it, 4 ways: 
* pip install tox - not working (not permission to install)
* pip install --user tox - not working (tox command not found on next command)
* sudo pip install --user tox - not working tox command not found
* sudo pip install tox - is only working option

Comment (swilly22) [reply .circleci/config.yml:33]: I think it should be `pip install tox --user`

**Comment by boris-42** [comment]
@swilly22 okay i did fix tox installation, I can try to fix tomorrow functional tests so we have CI green again

**Review by swilly22 — APPROVED** [review]

**Review by swilly22 — APPROVED** [review]

**Pull request merged**